src/Reflection/ReflectionAttribute.php

```diff
@@ -11,19 +11,31 @@
 use Roave\BetterReflection\Reflection\StringCast\ReflectionAttributeStringCast;
 use Roave\BetterReflection\Reflector\Reflector;
 
+use function array_map;
+
 class ReflectionAttribute
 {
+    private string $name;
```

Comment:
Should we have `non-empty-string` on this?

Reply:
Done.